docker-compose.yml

```diff
@@ -3,7 +3,8 @@
 version: "3.0"
 services:
   little-link:
-    image: ghcr.io/techno-tim/littlelink-server:latest
+    image: docker.io/library/links   
```

Comment:
I think this might be something you were testing?

Reply:
I always forget that haha